### PR TITLE
Set Cassandra listen_address and rpc_address

### PR DESCRIPTION
**Pull request by marko-asplund**
Set Cassandra listen_address and rpc_address param values during installation #1382

### CODE REVIEW

**Comment by marko-asplund** [comment]
@msmith-techempower @hamiltont This is a _bug fix only_ PR that should take care of the Cassandra installation problems. Please review.

File: toolset/setup/linux/installer.py [hunk 1]
```diff
@@ -19,6 +19,14 @@ def install_software(self):
     linux_install_root = self.fwroot + "/toolset/setup/linux"
     imode = self.benchmarker.install
 
+    script_vars = {
+      'TFB_DBHOST': self.benchmarker.database_host
+    }
+    l=[]
+    for k,v in script_vars.iteritems():
+      l.append(k+"="+v)
```
Comment (hamiltont) [inline toolset/setup/linux/installer.py]: I'd vote to use export e.g. `l.append("export %s=%s" % (k,v))` - there's not much security risk and it ensures that even if there's a future change this variable will always be available

**Comment by hamiltont** [comment]
This LGTM. Mike mentioned in IRC that he's hoping to not take any more PRs, so I'm not sure if it's going to go in before R10

**Comment by marko-asplund** [comment]
@hamiltont I updated this PR to address your review comments.

@msmith-techempower I re-checked this PR after the change in a Vagrant development environment and it seems to work as before. Please consider merging this fix for R10.

**Comment by msmith-techempower** [comment]
We need this to work before R10 as cassandra is still giving us trouble.

**Pull request merged**